extensions/amp-form/0.1/form-dirtiness.js

```diff
@@ -29,17 +30,24 @@ const SUPPORTED_TYPES = {
 export class FormDirtiness {
   /**
    * @param {!HTMLFormElement} form
+   * @param {!Window} win
    */
-  constructor(form) {
+  constructor(form, win) {
```

Comment:
nit: don't forget to pass the new parameter passed into all of the constructor call-sites

Reply:
Thanks for the reminder :see_no_evil: it's not integrated into `amp-form` yet.